store/data_source.go

```diff
@@ -486,6 +486,23 @@ type DataSourceMessage struct {
 	AuthenticationDatabase string
 }
 
+type UpdateDataSourceMessage struct {
```

Comment:
exported: exported type UpdateDataSourceMessage should have comment or be unexported (revive)

Reply:
Done. Thank Danny Linter